### PR TITLE
Feature/actions

### PR DESCRIPTION
**Pull request by gayathri-pillai**
AddMoney,
SendMoney,
RequestMoney,
Login,
Register Components

### CODE REVIEW

**Comment by chris93007** [comment]
Add sreeraag and nandini to review.

File: client/starter-app/src/app/views/pages/add-money/add-money.component.html [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
-</p>
+<div class = "addmoney">
+
+<div class = "form">
+                <img id="image" class="addimage" src="assets/add.png" /> 
+
+    <div class = "formcontent">
+    <p class = "heading">
+<B>ADD MONEY TO WALLET</B></p>
+<form [formGroup]="addForm" (ngSubmit)="add(addForm.value, addForm.valid)">
```
Comment (nandini-vel) [inline client/starter-app/src/app/views/pages/add-money/add-money.component.html]: add(addForm) is sufficient

File: client/starter-app/src/app/views/pages/add-money/add-money.component.ts [hunk 2]
```diff
@@ -7,9 +9,26 @@ import { Component, OnInit } from '@angular/core';
...
+  add(value ,validity) {
+    
+    if(!validity){
+      alert("Please enter all fields correctly!");
+      return;
+      
+    }
+   
+    console.log(value);
+    console.log(this.addForm.value);
```
Comment (nandini-vel) [inline client/starter-app/src/app/views/pages/add-money/add-money.component.ts]: remove consoles before pushing to dev

File: client/starter-app/src/app/views/pages/login/login.component.scss [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+/*
+ * Card component
+ */
+.card {
+    background-color: #34495E;
+    /* just in case there no content*/
+    padding: 20px 25px 30px;
+    margin: 0 auto 25px;
+    margin-top: 50px;
```
Comment (nandini-vel) [inline client/starter-app/src/app/views/pages/login/login.component.scss:33]: you can combine margin and margin-top

File: client/starter-app/src/app/views/pages/login/login.component.ts [hunk 4]
```diff
@@ -7,9 +9,28 @@ import { Component, OnInit } from '@angular/core';
...
+  public loginForm = this.fb.group({
+    emailphone: ["",[ Validators.required, Validators.pattern("^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$|\\d{10}")]],
+    password: ["", [Validators.required, Validators.pattern("(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*[^a-zA-Z0-9]).{8,}")]]
+  });
+
+  constructor(public router : Router, public fb: FormBuilder) { }
+    
+  Login(value ,validity) {
+    if(!validity){
+      alert("Please enter all fields correctly!");
```
Comment (nandini-vel) [inline client/starter-app/src/app/views/pages/login/login.component.ts]: don't use alert boxes.

**Review by sree-mohan — APPROVED** [review]

**Pull request merged**